xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd

```diff
@@ -1059,6 +1104,21 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Whether equipment is suitable for cycles.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="RestStopDistance" type="LengthType" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>The maximum distance between two resting levels, in metres rounded to the nearest cm. It is recommended to install a rest platform at the bottom and top of each ramp, as well as every 10 metres in the case of a long ramp.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="SafetyEdge" type="SafetyEdgeEnumeration" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Whether there is a wheel guard (side border intended to block the wheel of the wheelchair to prevent falls)</xsd:documentation>
```

Comment:
```suggestion
					<xsd:documentation>Whether there is a wheel guard (side border intended to block the wheel of the wheelchair to prevent falls).</xsd:documentation>
```